### PR TITLE
Add top level command "npm run install-all"

### PR DESCRIPTION
**Pull request by nielm**
add command `npm run install-all`

This command installs the required node-modules in all the src directories and updating  all the package-lock.json files by executing the command `npm install --save` in each subdirectory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "eslint": "eslint .",
     "eslint-fix": "eslint --fix .",
     "typecheck": "tsc --project jsconfig.json --maxNodeModuleJsDepth 0 --noEmit",
-    "mdlint": "markdownlint '**/*.md' --config .mdl.json --ignore '**/node_modules/**' --ignore 'code-of-conduct.md'"
+    "mdlint": "markdownlint '**/*.md' --config .mdl.json --ignore '**/node_modules/**' --ignore 'code-of-conduct.md'",
+    "rebuild-package-lock": "find . -name package.json -a ! -path '*/node_modules/*' -execdir pwd \\;  -execdir npm install --save \\;"
```
Comment (henrybell) [inline package.json:7]: To do only the `package-lock.json` files you could do `npm install --package-lock-only` and skip the actual install? Or perhaps rename to `install-all`/`install-src-tree`? Or even add both.

**Review by henrybell — COMMENTED** [review]
Let me know what you think -- not a blocker.

File: package.json (hunk 1 above)
Comment (nielm) [reply package.json:7]: install-all makes more sense :)

Comment (henrybell) [reply package.json]: Nice -- possible also to update the commit message to match?

**Review by henrybell — APPROVED** [review]

**Pull request merged**